netlify.toml

```diff
@@ -0,0 +1,7 @@
+[build]
+  base = "userdocs/"
```

Comment:
Why not just leave `base` empty and `command = make -f Makefile.docs build-pages` ?

Reply:
Netlify tried to be magic and installs `requirements.txt` because it finds it in base. Ill try installing explicitly instead

Reply:
ah I see

Reply:
I just saw it also needs `runtime.txt` in the base dir so I'll leave it as `userdocs`